lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -292,9 +292,23 @@ func (handler *StorageHandlersImpl) CreateVolume(params storage.CreateVolumePara
 }
 
 //RemoveVolume : Remove a Volume from existence
-func (handler *StorageHandlersImpl) RemoveVolume(storage.RemoveVolumeParams) middleware.Responder {
+func (handler *StorageHandlersImpl) RemoveVolume(params storage.RemoveVolumeParams) middleware.Responder {
 	defer trace.End(trace.Begin("storage_handlers.RemoveVolume"))
-	return storage.NewRemoveVolumeOK() //TODO: this is just a stub for now.
+
+	_, err := storageVolumeLayer.VolumeGet(context.TODO(), params.Name)
```

Comment:
alright yeah, I will change this. This will really just be calling `VolumeDestroy` and then type switching over the errors. pretty simple fix.